torch/csrc/jit/script/function_schema_parser.cpp

```diff
@@ -58,7 +59,9 @@ struct SchemaParser {
     });
     idx = 0;
     L.expect(TK_ARROW);
-    if (L.cur().kind == '(') {
+    if (L.nextIf(TK_DOTS)) {
```

Comment:
`... foo()` does look a bit weird, shall we use `(...) foo` instead?

Reply:
I would support both then :-)